packages/base/Tag/src/Tag/Tag.tsx

```diff
@@ -35,91 +35,109 @@ export interface Props extends BaseProps, TextLabelProps, DivOrAnchorProps {
-  variant?: 'light-grey' | 'blue' | 'green' | 'yellow' | 'red'
+  variant?: Variant
   /** ReactNode rendered after label */
   endAdornment?: ReactNode
   hovered?: boolean
 }
 
-const useStyles = makeStyles<Theme>(styles, { name: 'PicassoLabel' })
-
-// eslint-disable-next-line react/display-name
 export const Tag = forwardRef<HTMLDivElement, Props>(function Tag(props, ref) {
   const {
-    as = 'div',
+    as: Root = 'div',
     className,
     disabled,
     endAdornment,
-    hovered,
     children,
     icon,
     onDelete,
+    onClick,
     style,
     titleCase: propsTitleCase,
     variant = 'light-grey',
+    role,
+    // `hovered` is extracted so we don't pass it to Root with {...rest}
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    hovered,
     ...rest
   } = props
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { color, ...htmlAttributes } = rest
-  const classes = useStyles()
-
   const titleCase = useTitleCase(propsTitleCase)
 
-  const label = (
-    <>
-      <span className={classes.innerLabel}>
-        {titleCase ? toTitleCase(children) : children}
-      </span>
-      {endAdornment}
-    </>
-  )
-
   const handleDelete = (event: MouseEvent) => {
     if (disabled) {
       return
     }
 
     if (onDelete) {
       event.preventDefault()
+
+      /**
+       * This comment prevents the options list from opening.
+       * If the event propagates, the component interprets it
+       * as a regular click, which triggers the opening of the
+       * options list.
+       */
+      event.stopPropagation()
+
       onDelete()
     }
   }
 
   return (
-    <Chip
-      {...htmlAttributes}
+    <Root
+      role={role || (onDelete || onClick ? 'button' : undefined)}
+      aria-disabled={disabled}
       ref={ref}
-      classes={{
-        root: classes.root,
-        label: classes.label,
-        clickable: classes.clickable,
-      }}
-      className={cx(className, classes[variant], {
-        [classes.hovered]: hovered,
-        [classes.disabled]: disabled,
-      })}
+      className={twMerge(
+        `transition-none border-[0.041625rem] border-solid rounded-[6.25rem]
+          h-6 max-w-full inline-flex justify-center items-center cursor-default bg-white
+          group align-middle`,
+        classByVariant[variant],
+        className,
+        disabled ? 'text-gray-500 border-gray-200 pointer-events-none' : ''
+      )}
       style={style}
-      icon={
-        icon
-          ? React.cloneElement(icon, {
-              color: disabled ? 'grey' : 'dark-grey',
-            })
-          : undefined
-      }
-      label={label}
-      deleteIcon={
+      onClick={onClick}
+      tabIndex={onDelete || onClick ? 0 : undefined}
+      {...rest}
+    >
+      {icon && (
+        <span
+          className={twMerge(
+            'w-min h-min flex items-center -mr-1 ml-3',
+            disabled ? '@text-gray-200' : 'text-graphite-700'
```

Comment:
```suggestion
            disabled ? 'text-gray-200' : 'text-graphite-700'
```
I guess this is just typo